stingray/powerspectrum.py

```diff
@@ -204,7 +204,7 @@ def compute_rms(self, min_freq, max_freq, white_noise_offset=0.):
 
         if self.norm.lower() == 'leahy':
             powers_leahy = powers.copy()
-        elif self.norm.lower() == "frac":
+        elif self.norm.lower() in ["frac", "abs", "none"]:
```

Comment:
After all, this should be general enough to allow for any normalization, even the invalid ones. So, maybe, this could be transformed into an `else` (eliminating the following error)

Reply:
Ok, I changed it